docs/src/benchmarks/index.md

```diff
@@ -0,0 +1,101 @@
+
+The following hardware and software specifications describe the environment in which these benchmarks are measured. The device runs ThinEdge.io and its associated plugins for data collection.
+
+| Component      | Specification                                                 |
+|----------------|---------------------------------------------------------------|
+| **Processor**  | Quad-core ARM Cortex-A72 (ARM v8)                             |
+| **Memory**     | 1.8 GB RAM                                                    |
+| **Operating System** | Debian GNU/Linux 12 (bookworm)                          |
+| **MQTT Broker**| Mosquitto (v2.0.18)                                           |
+| **Firmware**   | ThinEdge.io (v1.1.2)                                          |
+| **Plugins Used** | `tedgecpuprocent`, `memory`, `tedge-agent`, `tedge-mapper`  |
+
+> The device is a Raspberry Pi 4 Model B Rev 1.5, a member of the OSADL QA Farm, which continuously collects data on performance and resource consumption.
+
+## CPU Run-time Consumption of Key ThinEdge.io Tasks
+
+In this section, we monitor the CPU usage of the most critical ThinEdge.io tasks: `mosquitto`, `tedge-agent`, and `tedge-mapper`. These tasks are essential for device-to-cloud communication, and their performance is crucial for the overall efficiency of the system.
+
+### Graph of CPU Usage
+
+Below is a visual representation of the CPU consumption by the main ThinEdge.io processes over the past month.
+
+![CPU Usage](./tedgecpuprocent-month.svg)
+
+For each process, the graph shows:
+- **Command Name (comm)**: The name of the task running on the device.
+- **CPU Usage (cuc)**: The percentage of CPU time utilized by each process.
+
+
+The graph allows you to visually track CPU performance trends and identify potential bottlenecks in the system.
+
+### Detailed Metrics
+
+The following table provides a detailed description of the CPU consumption of measured tasks:
+
+| Task                 | Description                                                    |
+|----------------------|----------------------------------------------------------------|
+| `mosquitto`          | Handles MQTT communications between devices and cloud          |
+| `tedge-agent`        | Coordinates device management, software updates, and telemetry |
+| `tedge-mapper_c8y`   | Converts telemetry data into the format required by cloud      |
+| `tedge-mapper-colle` | Converts telemetry data into the format required               |
+
+> Note: The values in this table are updated dynamically once per month.
+
+
+## Memory Usage
+
+In this section, we monitor the memory consumption of key system components to ensure efficient performance. The memory plugin tracks several metrics to provide a detailed view of how the system memory is being utilized.
+
+### Graph of Memory Usage
+
+Below is a visual representation of the memory consumption on the device over the past month.
+
+![Memory Usage](./memory-month.svg)
```

Comment:
The memory script should only show the thin-edge.io applications and not the whole OS memory usage etc.